### DISCUSSION TITLE
v5有适配win10的配置吗？用wiki的模版也无法运行报错。

### DISCUSSION THREAD

**Discussion by chinaniupai** [IrineSistiana/mosdns#661] [Q&A]
根本没有错误日志

**Comment by Mrican** [comment]
可尝试我的 https://github.com/Mrican/mosdns-config

放在与mosdns同目录下，双击service-install.bat，看到running即可

**Reply by chinaniupai** [reply]
> 可以试试我的https://github.com/Mrican/mosdns-config
> 
> 放与mosdns同目录下，双击service-install.bat，看到运行即可

谢谢你，我现在用了AdGuard Home，刚好满足我需要缓存功能和去广告功能。
至于分流功能，我只设置了一个外国的DNS，但是这个EDNS功能好像可以代替国内DNS分流的作用？
![image](https://user-images.githubusercontent.com/26068235/231401573-c1a59b1a-cd4d-4527-8c8d-1eacd782da07.png)
